src/raftstore/store/snap.rs

```diff
@@ -972,6 +972,7 @@ impl Snapshot for Snap {
                 let mut file = box_try!(File::open(&cf_file.path));
                 apply_plain_cf_file(&mut BufReader::new(file), &options, cf_handle)?;
             } else {
+                let _timer = INGEST_SST_DURATION_SECONDS.start_coarse_timer();
```

Comment:
I see you start this timer but I don't see you do other things to the timer (such as stop it).

Reply:
we will calculate the elapsed time in Drop